docs/source/isaura.rst

```diff
@@ -1,5 +1,160 @@
+
+.. _Isaura display:
+
+ .. image:: images/isaura/r8250_evt194237_dhits.png
+   :width: 46%
+ .. image:: images/isaura/r8250_evt194237_voxels.png
+   :width: 53%
+
+
+Seeing that events have already been divided into voxel-made 3D tracks, the next (and final) step comprises the computation of the **blobs** [#]_. This is a crucial stage within the reconstruction chain, since the energy of these elements will allow us to distinguish between double-electron (such as the double-beta signal) and single-electron (the majority of backgrounds) tracks.
+
+There are different ways to define the center of the blobs, the one selected by NEXT is illustrated in the figure below. According to this procedure, the two extreme voxels of the track are localized following the BFS algorithm, on a first stage. The energy-weighted averaged position of the hits inside the voxel will correspond to the **blob center**. From that point, a 3D sphere of radius ``blob_radius`` (specified in the config file) is taken, and all hits (independent to the voxel they belong to) inside that sphere will contribute to the energy of the blob.
+
+ .. image:: images/isaura/blobs_position_definition.png
+   :width: 400
+   :align: center
+
+
+To carry out this procedure, and as stated before, the first thing to do is to localize the two end voxels for each track. Defining the distance between any pair of voxels as the shortest path **along the track** that connects them, the two extreme voxels will be the ones with the longest distance between them. However, there are two special cases that are important to comment:
+
+
+ - It is possible that some spurious **low-energy** hits appear around the track (due to over-iterations during the *Richarson-Lucy* deconvolution process, as commented in :doc:`beersheba`; or some noise inside the chamber, for example). If these hits are reconstructed around the track but not far enough to produce a different S2 or track (taking into account the voxel size), they can be considered as a part of the main one and, being a bit separate, it is probable they end up belonging to an extreme voxel. That case would not be correct, and in order to solve it, the voxel will be dropped from the track and its energy passed to the closest one. This process is only carried out if the voxel energy is lower than: ``energy_threshold`` and the track is made by more than ``min_voxels`` voxels. Once this procedure is done, the extreme voxels are searched and found again recursively, until none of these conditions are fulfilled.
+
+ - Another particular scenario is the one that comes up when there are multiple end-voxel candidates (one can imagine that the shorter the track the more probable is this to happen). To deal with it, the more energetic candidates will be the ones set as extremes. With this convention, we aim to minimize the voxel-dropping algorithm commented just above.
```

Comment:
if you have two candidates for the extremes: one below the `energy_threshold` variable and the other one above that, the voxel dropping algorithm will not be performed, because the latter will be the one selected as the extreme voxel, according to the convention.

Reply:
I would state this explicitly.